open_event/static/js/admin/event/scheduling.js

```diff
@@ -0,0 +1,502 @@
+    currentUnscheduledStore = [];
+    $.each($unscheduledSessions, function (index, $sessionElement) {
+        $sessionElement = $($sessionElement);
+        var session = {
+            id: $sessionElement.data("session-id"),
+            title: $sessionElement.attr("data-original-text")
+        };
+        currentUnscheduledStore.push(session);
+    });
+}
+
+function sessionOverlapTest($track, $session) {
+    var $otherSessions = $track.find(".session.scheduled");
+    var returnVal;
+    returnVal = false;
+    $.each($otherSessions, function (index, $otherSession) {
+        $otherSession = $($otherSession);
+        if (!$otherSession.is($session) && collision($otherSession, $session)) {
+            returnVal = $otherSession;
+        }
+    });
+    return returnVal;
+}
+
+function restrictionCheck(x, $sessionElement) {
+    return (horizontallyBound($(".tracks"), $sessionElement, 0));
+}
+
+function overDraggable($sessionElement) {
+    return collision($(".tracks"), $sessionElement);
+}
+
+function updateColor($element) {
+    $element.css("background-color", palette.random("800"));
+    $element.css("background-color", palette.random("800"));
+}
+
+function updateElementTime($element) {
+    var topTime = moment.utc({hour: time.start.hours, minute: time.start.minutes});
+    var mins = pixelsToMinutes($element.outerHeight(false));
+    var topInterval = pixelsToMinutes($element.data("top"), true);
+
+    var startTimeString = topTime.add(topInterval, 'm').format("LT");
+    var endTimeString = topTime.add(mins, "m").format("LT");
+
+    $element.data("start-time", startTimeString).data("end-time", endTimeString);
+    $element.attr("data-original-title", startTimeString + " to " + endTimeString);
+    $element.tooltip("show");
+}
+
+function initializeInteractables() {
+
+    $tracks = $(".track");
+
+    interact(".session")
+        .draggable({
+            // enable inertial throwing
+            inertia: false,
+            // enable autoScroll
+            autoScroll: {
+                container: $(".track-container")[0],
+                margin: 50,
+                distance: 5,
+                interval: 10
+            },
+            restrict: {
+                restriction: ".draggable-holder"
+            },
+            // call this function on every dragmove event
+            onmove: function (event) {
+                var $sessionElement = $(event.target),
+                    x = (parseFloat($sessionElement.data('x')) || 0) + event.dx,
+                    y = (parseFloat($sessionElement.data('y')) || 0) + event.dy;
+
+                if (restrictionCheck(x, $sessionElement) || $sessionElement.hasClass("unscheduled")) {
+                    $sessionElement.css("-webkit-transform", "translate(" + x + "px, " + y + "px)");
+                    $sessionElement.css("transform", "translate(" + x + "px, " + y + "px)");
+
+                    $sessionElement.data('x', x);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2651583495/issues/source?bid=3233864&fileBranchId=3324174#l149)